tests/cases/test_enrich_ciaa_sources.py

```
@@ -0,0 +1,380 @@
+            "--skip-press-releases",
+            stdout=out,
+        )
+        output = out.getvalue()
+        assert "Found 1 DRAFT case(s)" in output
+
+
+class TestPressReleaseEnrichment:
+    def test_enriches_existing_pr_metadata(
+        self, data_dir, ag_index_csv, pr_index_csv, draft_case_with_press_release
+    ):
+        case, source = draft_case_with_press_release
```

Ruff (0.15.12)

[warning] Unpacked variable `case` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)